### PR TITLE
fix(blog): change to domain-less links

### PR DESCRIPTION
**Pull request by will-rice**
### PR Checklist

Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [x] I am requesting to **pull a topic/feature/bugfix branch** (right side). In other words, not _master_.
- [x] I have run `npm test` against my changes and tests pass.
- [x] I have added or edited necessary documentation, or no docs changes are needed.

### CODE REVIEW

File: content/blog/maker-models-on-embedded-devices.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 ---
-title: Learn to use wake word and text-to-speech on a Raspberry Pi.
-date: '2020-10-05'
+title: Learn to use Custom Wake Word and Text-to-Speech on a Raspberry Pi.
```
Comment (space-pope) [inline content/blog/maker-models-on-embedded-devices.md]:
```suggestion
title: Learn to Use Custom Wake Word and Text-to-Speech on a Raspberry Pi.
```

File: content/blog/maker-models-on-embedded-devices.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+hero: null
 ---
 
-One of the primary motivations for working on [spokestack-python](https://github.com/spokestack/spokestack-python) was to allow our models to run on embedded devices like [Raspberry Pi](raspberrypi.org). We are excited to show you how easy it is to use [Wake Word](/docs/concepts/wakeword) and [TTS](/docs/concepts/tts) models on these devices.
+One of the primary motivations for working on [spokestack-python](https://github.com/spokestack/spokestack-python) was to allow our models to run on embedded devices like [Raspberry Pi](https://www.raspberrypi.org/). We are excited to show you how easy it is to use [Wake Word](/docs/concepts/wakeword) and [TTS](/docs/concepts/tts) models on these devices.
 
 ## Spokestack Account
 
-You will want to [login](https://www.spokestack.io/account/login) and get your API keys for this tutorial. If you do not already have a Spokestack account please [create one](https://www.spokestack.io/account/create)
+You will want to [login](/account/login) and get your API keys for this tutorial. If you do not already have a Spokestack account please [create one](/account/create)
```
Comment (space-pope) [inline content/blog/maker-models-on-embedded-devices.md]:
```suggestion
You will want to [login](/account/login) and get your API keys for this tutorial. If you do not already have a Spokestack account, please [create one](/account/create).
```

**Review by space-pope — COMMENTED** [review]
I can't do line-by-line comments on lines not touched by this PR, so I'll note them here:

33: remove the comma after "In addition".
also 33: I might add a sentence here to the effect of "These instructions should all be followed while connected to your Raspberry Pi via SSH." 
53 (and following): for the dependencies section, I'd combine these into a single `apt-get` command (Raspbian's not using plain `apt` yet?) that the user can copy/paste and add the library names to the end of the preceding sentence: "...on the Raspberry Pi: PyAudio, Numpy, and Streamp3".
73: "TensorFlow's recommended.."
84: "...through [or "via"] `pip`"
also 84: I'd leave out the last sentence. Versions before 1.0.0 are understood to have APIs subject to change.
92: I'd leave out "Spokestack" and expand the link to read "[Hello, World! project](LINK)".
also 92: "Keep in mind..." -> "We've already installed the dependencies, so you won't need to follow that section in the project's readme."
107: "...one of our [support channels](/support)."

File: content/blog/maker-models-on-embedded-devices.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ This tutorial is geared toward the [Raspberry Pi 4B](https://www.raspberrypi.org
 
 ## Raspberry Pi Setup
 
-For the initial setup of the Raspberry Pi we recommend following the [Adafruit Voice Bonnet tutorial](https://learn.adafruit.com/adafruit-voice-bonnet/overview). This guide walks you through everything from OS installation to sound configuration. In addition, to the Adafruit instructions there are few Spokestack specific instructions/tips in the following.
+For the initial setup of the Raspberry Pi we recommend following the [Adafruit Voice Bonnet tutorial](https://learn.adafruit.com/adafruit-voice-bonnet/overview). This guide walks you through everything from OS installation to sound configuration. In addition to the Adafruit instructions there are few Spokestack specific instructions/tips in the following. These instructions should be followed while connected to your Raspberry Pi via SSH.
```
Comment (space-pope) [inline content/blog/maker-models-on-embedded-devices.md]:
```suggestion
For the initial setup of the Raspberry Pi we recommend following the [Adafruit Voice Bonnet tutorial](https://learn.adafruit.com/adafruit-voice-bonnet/overview). This guide walks you through everything from OS installation to sound configuration. In addition to the Adafruit instructions, there are a few Spokestack-specific instructions/tips in the following. These instructions should be followed while connected to your Raspberry Pi via SSH.
```
Sorry; missed the "a" in my first reading. Figured I'd add a bit of punctuation while I was here.

File: content/blog/maker-models-on-embedded-devices.md [hunk 4]
```diff
@@ -81,15 +71,15 @@ sudo apt-get install python3-tflite-runtime
...
+Spokestack should be installed through `pip`. We are currently using `v0.0.20` for this tutorial.
 
 ```shell
 pip install spokestack==0.0.20
 ```
 
 ## Testing with a Project
 
-We will test with our Spokestack "Hello, World!" [project](https://github.com/spokestack/python-hello-world). Keep in mind we installed the dependencies in the previous sections.
+We will test with our "[Hello, World! project](https://github.com/spokestack/python-hello-world)". Keep in mind we installed the dependencies in the previous sections so you will not need to follow that project's README.
```
Comment (space-pope) [inline content/blog/maker-models-on-embedded-devices.md]:
```suggestion
We will test with our ["Hello, World!" project](https://github.com/spokestack/python-hello-world). Keep in mind we installed the dependencies in the previous sections, so you will not need to follow that project's README.
```

File: content/blog/maker-models-on-embedded-devices.md [hunk 5]
```diff
@@ -1,18 +1,18 @@
...
-title: Learn to use wake word and text-to-speech on a Raspberry Pi.
-date: '2020-10-05'
+title: Learn to Use Custom Wake Word and Text-to-Speech on a Raspberry Pi.
+date: '2021-04-28'
 description: Following this guide teaches you how to deploy your Maker models on an embedded device.
 author: will
 tags: Apps, Engineering, Python, Tutorial
 draft: true
-hero: ../assets/blog/
+hero: null
```
Comment (timmywil) [inline content/blog/maker-models-on-embedded-devices.md]:
```suggestion
```

**Pull request merged**